src/coreclr/src/System.Private.CoreLib/src/ShouldSkipLocalsInit.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Used to indicate to the compiler that the .locals init flag should
+// not be set in nested method headers when emitting to metadata.
+[module: System.Runtime.CompilerServices.SkipLocalsInit]
```

Comment:
It's a small file, but why do we need three identical copies? We have other files that are used in corelib and other libraries.